clients/download.go

```diff
@@ -50,9 +50,7 @@ func DownloadPluginFromGithub(ctx context.Context, localPath string, org string,
 	if org != "cloudquery" {
 		pathInArchive = fmt.Sprintf("cq-%s-%s", typ, name)
 	}
-	if runtime.GOOS == "windows" {
-		pathInArchive += ".exe"
-	}
+	pathInArchive = withBinarySuffix(pathInArchive)
```

Comment:
I think we just needed it here so no need to do `withBinarySuffix`

Reply:
We already have it here and it still fails (see failure in other comment)